docs/solutions/sql_server_always_on_availability_groups/modules/instances/Sql/main.tf

```diff
@@ -0,0 +1,54 @@
+resource "oci_core_instance" "instance" {
+  count               = "${var.ad_count}"
+  availability_domain = "${lookup(data.oci_identity_availability_domains.ADs.availability_domains["${count.index}"],"name")}"
+  compartment_id      = "${var.compartment_id}"
+  display_name        = "${var.dns_label}${"${count.index}" + 1}"
+
+  #hostname_label      = "${var.dns_label}${"${count.index}" + 1}"
+  hostname_label = ""
+  image          = "${var.image_id}"
+  shape          = "${var.shape}"
+
+  create_vnic_details {
+    subnet_id        = "${var.subnets["${count.index}"]}"
+    assign_public_ip = false
+  }
+
+  timeouts {
+    create = "60m"
+  }
+
+  provisioner "local-exec" {
+    command = "sleep 10"
```

Comment:
What's the reason for this?

Reply:
We found that TF would fail at times due to the order in which things are built. This solves this

Reply:
I've opened an internal bug to track this - this should not be required.